### PR TITLE
build: cleanup and document instruments build

### PR DESCRIPTION
**Pull request by Benjozork**
Part of #5107

## Summary of Changes

* Make `src/instruments/buildSrc/igniter/tasks.mjs` provide both rollup inputs and igniter tasks
* Remove old devserver files (moved to [flybywiresim/ace](https://github.com/flybywiresim/ace))
* Document build process in `src/instruments/README.md`

Discord username (if different from GitHub): someperson#4953

## Testing instructions

* Make sure avionics still work

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by Benjozork** [comment]
Need to see if build works properly.

**Review by tracernz — APPROVED** [review]

**Comment by marcman86** [comment]
Quality Assurance Tester Report
Discord:marcman86#4907
Object of testing: #5108
Tier of Testing: 1
Date: 23/07/2021

Testing Process:
- Spawned on runway at EDDP
- Filled in missing values in the MCDU
- Took off
- Tested flight controls, AP (FCU) and avionics like ECAM, displays and lights

Negatives: -

Testing Results: Passed
Conclusions: Short flight. All systems behave like normal.

Hope this helps :-)

**Comment by TheSquirtingPantin** [comment]
Quality Assurance Tester

Discord: Remi#8871
Object of testing: #5108
Tier of Testing: 2
Date: 24/07/2021

Testing Process:
1) Full Flight KMIA to KIAD
2) Checked avionic behavior (FCU, DU, MCDU, AP functionalities...)

Negatives: N/A

Conclusions: PASS. All avionics working as expected
Media: N/A

File: src/instruments/buildSrc/igniter/tasks.mjs [hunk 1]
```diff
@@ -34,9 +34,19 @@ const ecamPages = [
...
+
+    return [
+        ...baseInstruments.map(({ name }) => ({ path: name, name, isInstrument: true })),
+        ...ecamPages.map((def) => ({ ...def, isInstrument: false })),
+    ];
+}
+
+export function getInstrumentsIgniterTasks() {
+    const baseInstruments = fs.readdirSync(`${Directories.instruments}/src`, { withFileTypes: true })
+        .filter((d) => d.isDirectory() && fs.existsSync(`${Directories.instruments}/src/${d.name}/config.json`));
```
Comment (davidwalschots) [inline src/instruments/buildSrc/igniter/tasks.mjs:49]: **issue**: This breaks building stuff on Windows.
**suggestion**: Please use `path.join` as per #5278 (and the code before you made the change).

Comment (tracernz) [reply src/instruments/buildSrc/igniter/tasks.mjs:49]: If we really want to support it we need CI for it.

Comment (davidwalschots) [reply src/instruments/buildSrc/igniter/tasks.mjs:49]: Without it using watch mode on Windows is just not going to work.
Using `path.join` is also just good practice which should always be applied to nodejs code 😀.

Comment (davidwalschots) [reply src/instruments/buildSrc/igniter/tasks.mjs:49]: Plus, I made all the changes to make it work. So it's just nice if it's not broken again.

Comment (tracernz) [reply src/instruments/buildSrc/igniter/tasks.mjs:49]: Yeah, I understand that. Just saying that we should look at CI so it doesn't get broken accidentally again. :)

**Pull request merged**